logs_ingest/main.py

```diff
@@ -89,6 +90,8 @@ def process_logs(events: List[func.EventHubEvent], self_monitoring: SelfMonitori
 
 
 def process_record(dt_payload: List[Dict], record: Dict, self_monitoring: SelfMonitoring):
+    if re.findall('[0-9]{2}/[0-9]{2}/[0-9]{4} [0-9]{2}:[0-9]{2}:[0-9]{2}', record["time"]):
+        record["time"] = str(datetime.strptime(record["time"], '%m/%d/%Y %H:%M:%S').isoformat()) + "Z"
```

Comment:
I'd rather extract this conversion into a separate method and move the call to the `main.parse_record `method (where we do the parsing).